sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/ChatThreadAsyncClient.java

```diff
@@ -88,32 +88,35 @@ public String getChatThreadId() {
     /**
      * Updates a thread's properties.
      *
-     * @param options Options for updating a chat thread.
+     * @param topic The new topic.
      * @return the completion.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Response<Void>> updateChatThreadWithResponse(UpdateChatThreadOptions options) {
+    public Mono<Response<Void>> updateTopicWithResponse(String topic) {
         try {
-            return withContext(context -> updateChatThread(options, context));
+            return withContext(context -> updateTopic(topic, context));
         } catch (RuntimeException ex) {
 
             return monoError(logger, ex);
         }
     }
 
     /**
-     * Updates a thread's properties.
+     * Updates a thread's topic.
      *
-     * @param options Options for updating a chat thread.
+     * @param topic The new topic.
      * @param context The context to associate with this operation.
      * @return the completion.
      */
-    Mono<Response<Void>> updateChatThread(UpdateChatThreadOptions options, Context context) {
+    Mono<Response<Void>> updateTopic(String topic, Context context) {
         context = context == null ? Context.NONE : context;
 
-        Objects.requireNonNull(options, "'options' cannot be null.");
```

Comment:
Should we require the new topic to be not null?

Reply:
Previously it could've been `null` within the `options` object.  I don't think the API has a problem with a `null` topic?

Reply:
the consensus was to not have any validations on the client code in general and leave the API handle it.

Reply:
@danielgerlag sorry I am late on this one. The consensus in general was to not add any client side validations and let the API handle it (that applies for most of the end points).